src/axios/auth.js

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+import qs from 'qs';
+
+export const login = (email, password, setCookie) => {
+  const options = {
+    method: 'POST',
+    headers: { 'content-type': 'application/x-www-form-urlencoded' },
+
+    data: qs.stringify({
+      email: `${email}`,
+      password: `${password}`,
+    }),
```

Comment:
신박한 방법이네요. 다만 여기서는 자바스크립트 특성상 이렇게만 써도 괜찮답니다!
```suggestion
    data: qs.stringify({
      email,
      password
    }),
```

Reply:
아..그렇군요 감사합니다!!